ingest/ledgerbackend/stellar_core_runner.go

```
@@ -96,24 +96,33 @@ func (r *stellarCoreRunner) generateConfig() string {
 		"UNSAFE_QUORUM=true",
 		fmt.Sprintf(`NETWORK_PASSPHRASE="%s"`, r.networkPassphrase),
 		fmt.Sprintf(`BUCKET_DIR_PATH="%s"`, filepath.Join(r.tempDir, "buckets")),
+		fmt.Sprintf(`HTTP_PORT=%d`, r.httpPort),
 	}
 	for i, val := range r.historyURLs {
 		lines = append(lines, fmt.Sprintf("[HISTORY.h%d]", i))
 		lines = append(lines, fmt.Sprintf(`get="curl -sf %s/{0} -o {1}"`, val))
 	}
-	// Add a fictional quorum -- necessary to convince core to start up;
-	// but not used at all for our purposes. Pubkey here is just random.
-	lines = append(lines,
-		"[QUORUM_SET]",
-		"THRESHOLD_PERCENT=100",
-		`VALIDATORS=["GCZBOIAY4HLKAJVNJORXZOZRAY2BJDBZHKPBHZCRAIUR5IHC2UHBGCQR"]`)
-	return strings.ReplaceAll(strings.Join(lines, "\n"), "\\", "\\\\")
+	if r.coreConfigAddendumPath == "" {
```

Can we return an error here if `generateConfig` was called from `runFrom` (so we expect to have quorum slice set by user)? I think that right now it will just create _catchup_ config.

---

The caller ensures that the path isn't empty in online mode.

---

OK, I can see it now but I think `stellarCoreRunner` should do it because it's their responsibility. It's possible that it will be used outside of `CaptiveStellarCore` in the future and this will generate invalid config files.

---

Fair enough. Then we should pass online/offline as a parameter. Are you OK with that?

---

Sounds good.